README.md

```diff
@@ -29,3 +29,7 @@ Vale always runs from the workspace directory in either case, so if you put a [V
 Currently this extension only supports **Markdown, reStructuredText, LaTeX and plain text (.txt) documents**; please [open an issue][issue] or a pull request if you need support for **more document formats**, provided that [Vale][] supports them.
 
 [config]: https://errata-ai.github.io/vale/config/
+
+## Configuration
+
+- `vscode-vale.path`: Specifies the path to the `vale` executable, useful if you don't want to use the global binary. The path should be relative to the workspace root folder.
```

Comment:
@emmenko It could be me, but I am trying this branch, creating a package of the extension and loading it into VSC, but it's not recognising the setting. What steps did you take to check this locally?

Unfortunately there seem to be a myriad different ways with VSC :/

Reply:
I added the config in my workspace settings, then I used the "Start debugging mode" with `F5`.

What would be the "proper way" to test this? It's the first time I work with a VSCode extension, so I'm not familiar with the usual development process.

Reply:
Ah, in workspace settings, let me see.

Reply:
I don't know, I can't get this to run at all. Here's what I propose. I'll merge this and release, then could you make another PR with an update to the read me, telling people *exactly* where to add the config, it's confusing as VSC has about 4 places for settings…

Reply:
Sounds like a plan. Cool thanks, we'll try it out and I'll update the docs if everything works fine 👌

Reply:
OK, it's published, might take a little time to find it's way through the web. Then take a whirl and anything you can add to the docs is appreciated

Thanks for your contribution @emmenko !